File: Testing/gtm001.js
// load gtm module
var gtm = require('gtm');

// create gtm database connection
var db = new gtm.Database();

console.log("\n");
console.log('Node.js Version: ' + process.version);

console.log('Version: ' + db.version());

console.log('About: ' + db.about());

// Set a global
db.set('^Person',"John Lennon");

// Get the global
var name = db.get('^Person');

console.log('^Person '+name);

// Kill the global
db.kill('^Person');

db.execute('write $ZVERSION,!');

File: Testing/gtm001.py
# load gtm module
from gtm import GTM

db = GTM()

print db.About()

print db.Version()

db.Set("^Capital","Washington")

value = db.Get("^Capital")

print value

db.Kill("^Capital")

db.Set('^Person(1,"name",)',"John Lennon");
db.Set('^Person(1,"address")',"Abby Road");

print db.Query('^Person(1)')

print db.Order('^Person(1,"address")')

# help(db)
